org.eclipse.jdt.launching/launching/org/eclipse/jdt/launching/JavaRuntime.java

```diff
@@ -115,102 +114,85 @@
 import org.xml.sax.SAXException;
 
 /**
- * The central access point for launching support. This class manages
- * the registered VM types contributed through the
- * <code>"org.eclipse.jdt.launching.vmType"</code> extension point.
- * As well, this class provides VM install change notification,
- * and computes class paths and source lookup paths for launch
- * configurations.
+ * The central access point for launching support. This class manages the registered VM types contributed through the
```

Comment:
Manu unrelated format changes in JavaRuntime.java

Reply:
Same case as of JavaLaunchableTester.java - So removed the changes now.